src/System.Net.Http/src/System/Net/Http/HttpRequestException.cs

```diff
@@ -6,7 +6,6 @@
 
 namespace System.Net.Http
 {
-    [Serializable]
     [SuppressMessage("Microsoft.Serialization", "CA2229")]
```

Comment:
I believe that suppression is still valid because it's a warning that the type implements ISerializable and doesn't have a serialization constructor.